funnel/forms/space.py

```diff
@@ -2,13 +2,15 @@
 
 import re
 from coaster.utils import sorted_timezones
+from wtforms.widgets import CheckboxInput, ListWidget
+from wtforms.ext.sqlalchemy.fields import QuerySelectMultipleField
```

Comment:
I think we should expose the SQLAlchemy fields in Baseframe forms so they don't need specific import here.